### ISSUE TITLE
Changing versioning for our packages for the beta release:

### ISSUE THREAD

**Issue by bobstrecansky** [open-telemetry/opentelemetry-php#837]
Shamelessly stolen from @Nevay 👍🏼 

We likely have to change our versioning for beta (we currently use the monorepo tags for every subsplit, right?) - packages should be versioned independently, we don't want to bump our API version for unrelated SDK changes ([spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#version-numbers)).

**Comment by bobstrecansky** [comment]
@Nevay - do you have an idea for an implementation strategy for this?

**Comment by brettmc** [comment]
One idea I had, that I haven't fully thought through: if we just git-split `main` into our read-only repos (or just don't tag in the monorepo), then we can instead tag in the subsplit repositories.
The downside of that is that we (maintainers) would need to identify everything that's changed and go tag it manually. So, more manual work and the possibility of missing tagging things (although I'm sure we could come up with a process, eg looking at diffs since last release)

**Comment by bobstrecansky** [comment]
Brett and I discussed the splitting of the API and the SDK as it pertains to tagging and releasing today.
https://github.com/open-telemetry/opentelemetry-php/issues/837


We are semi-conflicted about the next steps that we should take.  We discussed not versioning the main repository, but rather versioning the git split other packages, namely the API and SDK read only packages.
If we decide to not version the main repository, we will be able to successfully release other packages independently, which is part of the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/16da729707e02e83f3a1b25b4faa3094c3f9e29b/specification/library-guidelines.md#version-labeling).  This may be confusing to end users that expect the version of the main repository to be current and up to date with the most recent release.


If we decide that we want to continue to use the versioning system in the main repository, we’ll have to make releases for the API and the SDK in a coupled fashion, which doesn’t follow spec.
Does anyone have experience with performing this action?  We were looking at prior art in the symfony and laravel repositories, but it was unclear what is the best path forward to being able to deliver separate portions of our codebase separately.  It does appear that both Java and Go are doing something similar to what we are doing now.
